argoverse/evaluation/detection/utils.py

```diff
@@ -157,8 +157,10 @@ def accumulate(
         logger.info(f"{dt_filtered.shape[0]} detections")
         logger.info(f"{gt_filtered.shape[0]} ground truth")
         if dt_filtered.shape[0] > 0:
-            ranked_detections, scores = rank(dt_filtered)
-            metrics = assign(ranked_detections, gt_filtered, cfg)
+            ranked_dts = rank(dt_filtered)
+            metrics = assign(ranked_dts, gt_filtered, cfg)
+
+            scores = [[record.score] for record in ranked_dts.tolist()]
```

Comment:
seems like this extra for-loop is less efficient than just returning the scores earlier, right?

Might be nice to avoid it

Reply:
Maybe we can still return the scores?

Reply:
Are you okay with the sizing mismatch where `ranked_dts` is `(N,)` and `scores` is `(N, 1)`?

Reply:
Is there a reason we need (N,1) here? If so, how about we return (N,) for both, and just reshape to (N,1) later where it's used?

Reply:
Resolved.